ubuntudesign/www.ubuntu.com
# fix links to third-party privacy policies

## Done

*  add a redirect from /privacypolicy/thirdparties to /legal/terms-and-policies/privacy-policy/third-parties
* update /legal/terms-and-policies/privacy-policy/third-parties to link to /legal/terms-and-policies/privacy-policy
* update /legal/terms-and-policies/privacy-policy to link to /legal/terms-and-policies/privacy-policy/third-parties
* update /legal/terms-and-policies/privacy-policy/2013-03-25 to link to  /legal/terms-and-policies/privacy-policy/third-parties
* update the relevant copy docs [privacy-policy](https://docs.google.com/document/d/1JaZomTAVAADxr1PT_1_4u2YBWxipGxqlbXjt3MCDrsY/edit), [privacy-policy/2013-03-25](https://docs.google.com/document/d/1bDDY0NU8MybiMjso2mxVtoJl8Phgf98JtPa0b0pE2MI/edit) and [privacy-policy/third-parties](https://docs.google.com/document/d/1o4XTDcTr89_7-SalMR-tkcyusS51zDRAhAWC_A8g_pQ/edit)

## QA

1. go to /legal/terms-and-policies/privacy-policy and see that it links to /legal/terms-and-policies/privacy-policy/third-parties in the text
2. go to /legal/terms-and-policies/privacy-policy/2013-03-25 and see that it links to /legal/terms-and-policies/privacy-policy/third-parties in the text
3. go to /aboutus/privacypolicy/thirdparties and see that it redirects to /legal/terms-and-policies/privacy-policy/third-parties
4. go to /privacypolicy/thirdparties/ and see that it redirects to /legal/terms-and-policies/privacy-policy/third-parties
5. go to /privacy-policy/thirdparties/ and see that it redirects to /legal/terms-and-policies/privacy-policy/third-parties

## Issue / Card

Fixes #1390
